### PR TITLE
Update model.py

### PR DESCRIPTION
**Pull request by PrathamSoni**
Fixes the bottom right corner error for testing process.

### CODE REVIEW

File: model.py [hunk 1]
```diff
@@ -154,6 +154,12 @@ def predict(self,x):
 				for j in range(num_down):
 					tmp = self.sess.run(self.out,feed_dict={self.input:[x[i*self.img_size:(i+1)*self.img_size,j*self.img_size:(j+1)*self.img_size]]})[0]
 					tmp_image[i*tmp.shape[0]:(i+1)*tmp.shape[0],j*tmp.shape[1]:(j+1)*tmp.shape[1]] = tmp
+			#this added section fixes bottom right corner when testing
+			if (x.shape[0]%self.img_size != 0 and  x.shape[1]%self.img_size != 0):
+				tmp = self.sess.run(self.out,feed_dict={self.input:[x[-1*self.img_size:,-1*self.img_size:]]})[0]
+				tmp_image[-1*tmp.shape[0]:,-1*tmp.shape[1]:] = tmp
+			scipy.misc.imsave('test1.png', tmp_image)
```
Comment (jmiller656) [inline model.py]: Hey, thanks for your work. Not sure why this line is needed though...

**Review by jmiller656 — CHANGES_REQUESTED** [review]

File: model.py (hunk 1 above)
Comment (PrathamSoni) [reply model.py]: The line is necessary to fix the issue where the bottom right corner is not computed. Your existing code neglects to compute this area and this code fixes it.

Comment (jmiller656) [reply model.py]: I should have been more specific. I'm talking about line 161 in particular

Comment (PrathamSoni) [reply model.py]: oh that was just for me to save the image while testing the fix

Comment (jmiller656) [reply model.py]: Ok great. If you could remove that, I'll merge your pull request

**Comment by PrathamSoni** [comment]
removed the "debugging" save line.

**Review by jmiller656 — APPROVED** [review]

**Pull request merged**